tests/unit/scala/test_utils_suite.py

```diff
@@ -147,36 +147,35 @@ def check_zip_files_and_close_stream(input_stream, expected_files):
     with zip_file_or_directory_to_stream(
         test_files.test_udf_py_file,
         leading_path=test_files.test_udf_directory,
-        add_init_py=True,
     ) as stream:
         check_zip_files_and_close_stream(stream, ["test_udf_file.py"])
 
     with zip_file_or_directory_to_stream(
         test_files.test_udf_py_file,
         leading_path=os.path.dirname(test_files.test_udf_directory),
     ) as stream:
-        check_zip_files_and_close_stream(stream, ["test_udf_dir/test_udf_file.py"])
+        check_zip_files_and_close_stream(
+            stream, ["test_udf_dir/", "test_udf_dir/test_udf_file.py"]
+        )
 
     with zip_file_or_directory_to_stream(
```

Comment:
nit: it seems that this test case is now identical to the one above.

Reply:
good catch, done